mapbox-store-locator/ThemeViewController.swift

```diff
@@ -240,9 +241,11 @@ class ThemeViewController: UIViewController, MGLMapViewDelegate, CLLocationManag
                 self.mapView.style?.addSource(source)
                 
                 let lineStyle = MGLLineStyleLayer(identifier: "route-style", source: source)
-                lineStyle.lineColor = MGLStyleValue(rawValue: (self.viewControllerTheme?.themeColor.navigationLineColor)!) // Set the line color to the theme's color.
-                lineStyle.lineJoin = MGLStyleValue(rawValue: NSValue(mglLineCap: .round))
-                lineStyle.lineWidth = MGLStyleValue(rawValue: 3)
+                
+                // Set the line color to the theme's color.
+                lineStyle.lineColor = NSExpression(forConstantValue: self.viewControllerTheme?.themeColor.navigationLineColor)
+                lineStyle.lineJoin = NSExpression(forConstantValue: NSValue(mglLineCap: .round))
```

Comment:
The `lineJoin` property is being set to an MGLLineCap, so this isn’t guaranteed to have the desired effect. Consider setting the property to a string constant of `round` instead.